parser-typechecker/src/Unison/Lexer.hs

```diff
@@ -34,6 +69,7 @@ data Err
   | InvalidEscapeCharacter Char
   | LayoutError
   | CloseWithoutMatchingOpen String String -- open, close
+  | Opaque String
```

Comment:
Is this just a convenient catch-all for when you would rather not make up a new data constructor and pipe it through the rest of the code?